phpunit.xml.dist

```diff
@@ -16,6 +16,7 @@
     timeoutForSmallTests="10"
     timeoutForMediumTests="20"
     timeoutForLargeTests="30"
+    executionOrder="defects"
```

Comment:
⬆️ ⬆️ ⬆️ ⬆️ ⬆️ 

Let's have it next to the other starting with `e`.

Reply:
no problem for me, but be aware that the list isn't sorted alphabetically anyway (and it doesn't make sense)